localstack/services/generic_proxy.py

```diff
@@ -24,10 +24,20 @@
 SERVER_CERT_PEM_FILE = '%s/server.test.pem' % (TMP_FOLDER)
 
 # CORS settings
+extra_cors_allowed_headers = os.environ.get('EXTRA_CORS_ALLOWED_HEADERS')
```

Comment:
Can we please put this into `config.py`. Ideally we would like to keep all configuration logic in a single place.